lib/runners/processor.rb

```diff
@@ -177,18 +177,35 @@ def add_warning_if_deprecated_version(minimum:, file: nil, deadline: nil)
       end
     end
 
-    def add_warning_if_deprecated_options(keys)
-      deprecated_keys = config_linter.slice(*keys).compact.keys
-        .map { |k| "- `#{config_field_path(k)}`" }
+    def add_warning_if_deprecated_options
+      name = :options
+      return unless config_linter[name]
 
-      unless deprecated_keys.empty?
-        add_warning <<~MSG, file: config.path_name
-          DEPRECATION WARNING!!!
-          The following options in your `#{config.path_name}` are deprecated and will be removed.
-          See #{analyzer_doc} for details.
-          #{deprecated_keys.join("\n")}
-        MSG
-      end
+      file = config.path_name
+      add_warning <<~MSG, file: file
+        DEPRECATION WARNING!!!
+        The `#{config_field_path(name)}` option is deprecated in your `#{file}`. Fix it as follows:
+        See #{analyzer_doc} for details.
+
+        ```diff
+         linter:
+           #{analyzer_id}:
+        -    options:
+        -      foo: "bar"
+        +    foo: "bar"
+        ```
+      MSG
+    end
+
+    def add_warning_for_deprecated_option(name, to:)
+      return unless config_linter[name]
+
+      file = config.path_name
+      add_warning <<~MSG, file: file
+        DEPRECATION WARNING!!!
+        The `#{config_field_path(name)}` option is deprecated in your `#{file}`. Use the `#{config_field_path(to)}` option instead.
```

Comment:
```suggestion
        The `#{config_field_path(name)}` option is deprecated. Use the `#{config_field_path(to)}` option instead.
```

Reply:
I doubt some users may not know what should be fixed if there is no `sider.yml` word in the warning message. 🤔

Reply:
For a better idea, we may need to improve the Web UI. We have not used the `file` value in a `warning` hash. 🤔

Reply:
If you want to add `#{file}` in the sentence, I guess the below is sufficient.

```suggestion
        The `#{config_field_path(name)}` option is deprecated in your `#{file}`. Use the `#{config_field_path(to)}` option instead in your `#{file}`.
```

Reply:
Do you mean to repeat `` in your `#{file}` ``?

Reply:
No, sorry, this is correct.
```suggestion
        The `#{config_field_path(name)}` option is deprecated. Use the `#{config_field_path(to)}` option instead  in your `#{file}`.
```

Reply:
Ah, sounds good. 👍